breese8009-bryan-reese/hw_01/app.js

```diff
@@ -0,0 +1,51 @@
+
+
+
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal. 
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+  
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+  
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else). 
+*/
+
+
+// using if else and for loop
+
+// 	for(var bottles=100; bottles>0; bottles--){
+
+//          bottles--;
```

Comment:
If you're dropping bottles 2 at a time, decrementing `bottles` twice would be why. 😝 

Each time this loops through, here's what happens:
* loop notes the value of `bottles`. it starts at 100.
* if `bottles` is more than 0, decrement the variable. `bottles` is now 99.
* repeat, so `bottles` starts at 99, and since it's still more than 0, it gets decremented to 98.